demos/src/themes.mustache

```diff
@@ -15,5 +15,75 @@
 		<div class='o-header-services__title'>
 			<h1 class='o-header-services__product-name'><a href="/">B2B Tool or Service name</a></h1><span class='o-header-subrand__product-tagline '>Tagline to explain the product here</span>
 		</div>
+	</div>	
+	<div class='o-header-services__primary-nav'>
+		<div class='o-header-services__container'>
+			<ul class='o-header-services__nav-list'>
+				<li class='o-header-services__nav-item'>
+					<a class='o-header-services__nav-link o-header-services__nav-link--selected' href='#'>
+						Nav item 1
+					</a>
+				</li>
+				<li class='o-header-services__nav-item'>
+					<a class='o-header-services__nav-link' href='#'>
+						Nav 2
+					</a>
+				</li>
+				<li class='o-header-services__nav-item'>
+					<a class='o-header-services__nav-link' href='#'>
+						Nav 3
+					</a>
+				</li>
+				<li class='o-header-services__nav-item'>
+					<a class='o-header-services__nav-link' href='#'>
+						Nav 4
+					</a>
+				</li>
+			</ul>
+		</div>
+	</div>
+	</header>
+	<div class="o-header__drawer--services o-header__drawer o--if-js" id="o-header-drawer" data-o-header-drawer data-o-header-drawer--no-js>
+	<div class="o-header__drawer-inner">
+		<div class="o-header__drawer-tools">
+			<span>Tool or Service name</span>
+			<button type="button" class="o-header__drawer-tools-close" aria-controls="o-header-drawer">
+				<span class="o-header__visually-hidden">Close</span>
+			</button>
+		</div>
+		<nav class="o-header__drawer-menu o-header__drawer-menu--primary" role="navigation" aria-label="Primary navigation">
+			<ul class="o-header__drawer-menu-list">
+				<li class='o-header__drawer-menu-item'>
+					<a class="o-header__drawer-menu-link" href='#'>
+						Nav item 1
+					</a>
+				</li>
+				<li class='o-header__drawer-menu-item'>
+					<a class="o-header__drawer-menu-link" href='#'>
+						Nav 2
+					</a>
+				</li>
+				<li class='o-header__drawer-menu-item'>
+					<a class="o-header__drawer-menu-link" href='#'>
+						Nav 3
+					</a>
+				</li>
+				<li class='o-header__drawer-menu-item'>
+					<a class="o-header__drawer-menu-link" href='#'>
+						Nav 4
+					</a>
+				</li>
+			</ul>
+		</nav>
+		<nav class="o-header__drawer-menu o-header__drawer-menu--user" role="navigation" aria-label="User navigation">
+			<ul class="o-header__drawer-menu-list">
+				<li class="o-header__drawer-menu-item">
+					<a class="o-header__drawer-menu-link" href="#">XXXX</a>
+				</li>
+				<li class="o-header__drawer-menu-item">
+					<a class="o-header__drawer-menu-link" href="#">Sign in</a>
+				</li>
+			</ul>
+		</nav>
+	</div>
 	</div>
```

Comment:
Can you add the nav into the b2c part of this code too so they are both the same? I'm worried that the omission of the nav from the b2c demo makes it look like only b2b is allowed a secondary nav.